actinia/utils.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#######
+# acintia-python-client is a python client for actinia - an open source REST
+# API for scalable, distributed, high performance processing of geographical
+# data that uses GRASS GIS for computational tasks.
+#
+# Copyright (c) 2022 mundialis GmbH & Co. KG
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+#######
+
+__license__ = "GPLv3"
+__author__ = "Anika Weinmann"
+__copyright__ = "Copyright 2022, mundialis GmbH & Co. KG"
+__maintainer__ = "Anika Weinmann"
+
+import json
+import requests
+import sys
+
+
+def request_and_check(url, auth, status_code=200):
+    resp = requests.get(url, auth=auth)
+    if resp.status_code != status_code:
+        raise Exception(f"Error {resp.status_code}: {resp.text}")
+    return json.loads(resp.text)
+
+
+def print_stdout(msg):
```

Comment:
A follow-up PR might introduce a real logging library, that can be turned on and off